jaypeasee/national-parkfinder
# TDD - Main Nav

## Is this a fix or a feature?
feature/testing

## What is the change?
Unit tests the following components:
* `MainNav`
* `ParkList`
* `ParkBtn`
* `NavSearch`

## Where should the reviewer start?
Review the test files for each of the above components.

## How should this be tested?
1. Look to see if I am missing any components that should be tested.
2. Confirm the tests are all passing with the command `npm test src/<folder name>/<component name>.test.js`

## Relevant Tickets
* #36 
* Closes #26
* Closes #23 
* Closes #22 
* #6 

@saraho1123
@jaypeasee
@baileydunning
